imod/formats/prj/prj.py

```diff
@@ -547,6 +547,15 @@ def _create_dataarray_from_paths(
         _read=imod.idf._read,
         headers=headers,
     )
+
+    # remove coordinates in factor and addition but not in da
+    if type(factor) is xr.DataArray:
+        for d in factor.dims:
+            if d not in da.dims:
+                if len(factor.coords[d]) == 1:
+                    factor = factor.isel({d: 0}).drop(d)
```

Comment:
`.isel({d: 0}, drop=True)` is probably what you're after?

Reply:
done